plugins/by-name/ltex-extra/default.nix

```
@@ -2,16 +2,24 @@
   lib,
   helpers,
   config,
+  options,
   ...
 }:
 with lib;
+let
+  lspCfg = config.plugins.lsp;
+in
 lib.nixvim.plugins.mkNeovimPlugin {
   name = "ltex-extra";
   packPathName = "ltex_extra.nvim";
   package = "ltex_extra-nvim";
 
   maintainers = [ maintainers.loicreynier ];
 
+  description = ''
+    This plugin works with both the ltex or ltex_plus language servers and will enable ltex_plus if neither are.
```

```suggestion
    This plugin works with either the ltex or ltex_plus language servers.
    Enabling this plugin will also enable ltex_plus, unless ltex is already enabled.
```

Is something like this clearer?